src/agent/onefuzz/src/syncdir.rs

```diff
@@ -265,9 +268,39 @@ impl SyncedDir {
                 if ignore_dotfiles && file_name_event_str.starts_with('.') {
                     continue;
                 }
-
                 event!(event.clone(); EventData::Path = file_name_event_str);
                 metric!(event.clone(); 1.0; EventData::Path = file_name_str_metric_str);
+                if let Some(jr_client) = jr_client {
+                    match event {
+                        Event::new_result => {
+                            jr_client
+                                .send_direct(
+                                    JobResultData::NewCrashingInput,
+                                    HashMap::from([("count".to_string(), 1.0)]),
+                                )
+                                .await;
+                        }
+                        Event::new_coverage => {
+                            jr_client
+                                .send_direct(
+                                    JobResultData::CoverageData,
+                                    HashMap::from([("count".to_string(), 1.0)]),
+                                )
+                                .await;
+                        }
+                        Event::new_crashdump => {
+                            jr_client
+                                .send_direct(
+                                    JobResultData::NewCrashDump,
+                                    HashMap::from([("count".to_string(), 1.0)]),
+                                )
+                                .await;
+                        }
+                        _ => {
+                            debug!("Unhandled job result!");
```

Comment:
We should upgrade this to at least the info level since we won't see debug logs in prod and we might otherwise miss the message (in case we forget to add a new case here)

Reply:
Same thing on line 360